client/src/App.scss

```diff
@@ -3,3 +3,14 @@
 body {
   padding-top: 5rem;
 }
+
+.nav-link {
```

Comment:
Override Bootstrap defined classes in App.scss. Then they will take effect across the entire app.